src/codec_aom.c

```diff
@@ -972,8 +972,9 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
             bps = 16;
         }
         aomImage.bps = bps;
-        aomImage.x_chroma_shift = alpha ? 1 : codec->internal->formatInfo.chromaShiftX;
-        aomImage.y_chroma_shift = alpha ? 1 : codec->internal->formatInfo.chromaShiftY;
+        // Monochrome is handled below. Use 4:2:0 for now.
+        aomImage.x_chroma_shift = (alpha || codec->internal->formatInfo.monochrome) ? 1 : codec->internal->formatInfo.chromaShiftX;
+        aomImage.y_chroma_shift = (alpha || codec->internal->formatInfo.monochrome) ? 1 : codec->internal->formatInfo.chromaShiftY;
```

Comment:
I think the changes to these two lines are not necessary, right?

Reply:
Technically correct. I am just applying the newly added comment `The chromaShiftX and chromaShiftY fields should be ignored when monochrome is AVIF_TRUE` for consistency.